<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player_Interface</class>
 <widget class="QWidget" name="Player_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Load">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="Music_Pos">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>780</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::handle:horizontal{ 
		width:  30px; 
		height: 30px; 
		margin-top: -10px; 
		margin-left: 0px; 
		margin-bottom: -10px; 
		margin-right: 0px; 
 		border-image:url(:/Resource/Rect.png);
} 
QSlider::sub-page:horizontal{
	   background:rgba(80,166,234,150);
}
QSlider::groove:horizontal{ 
		height: 12px; 
		left: 0px;
		right: 0px; 
		background:rgba(0,0,0,150);
 } </string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="PushButton" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>780</width>
     <height>60</height>
    </rect>
   </property>
   <widget class="QPushButton" name="Next">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Res.qrc">
      <normaloff>:/Resource/Next.png</normaloff>:/Resource/Next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Res.qrc">
      <normaloff>:/Resource/Back.png</normaloff>:/Resource/Back.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Play">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Res.qrc">
      <normaloff>:/Resource/Play.png</normaloff>:/Resource/Play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Audio_Show">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Res.qrc">
      <normaloff>:/Resource/Audio.png</normaloff>:/Resource/Audio.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="Audio_Size">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>20</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:vertical{ 
 		border-image:url(:/Resource/Rect.png);
} </string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Music_Text">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>340</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Music_Time">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Music_Image">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/Resource/Logo.png</normaloff>:/Resource/Logo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>90</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections/>
</ui>
